### PR TITLE
Changes to the User Interface

### PR DESCRIPTION
**Pull request by WhitneyOgutu**
*What does this PR do?* - New changes to the UI for all the pages and new landing page.

*Description of the PR* - The different pages has a new design with a header and footer included, the landing page (index.html) has additional sections as well.

*Background context* - The pages have no functionality, they just show the interface

*Screenshots*
jumbotron
description
footer
login
profile
register
register-business
reviews
search
testimonial
update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
-# UI-design
-User-interface design for WeConnect which is a platform that brings businesses and individuals together.
+WeConnect
+Andela Project Challage
```
Comment (ridgekimani) [inline README.md]: I think you can give more context on the weconnect short description rather than it's an Andela Project Challenge

**Comment by ridgekimani** [comment]
screen shot 2018-02-28 at 4 44 19 pm
The jumbotron is too large. I would also prefer you use white fonts...

**Comment by ridgekimani** [comment]
Then give some margin to the Heading "We Connect"

**Comment by ridgekimani** [comment]
Can you be able to navigate through the pages on your local machine?

**Comment by WhitneyOgutu** [comment]
@ridgekimani I've made all the changes above.
However, I'm having an issue with creating a new PR since I have made further changes to my branches

**Comment by WhitneyOgutu** [comment]
@ridgekimani I have already pushed to this branch though. Is there any need of creating a new PR ?

**Comment by Ogollah** [comment]
looks great

**Comment by ridgekimani** [comment]
@WhitneyOgutu you don't need to recreate a PR

**Pull request merged**